src/ds_caselaw_utils/courts.py

```diff
@@ -1,97 +1,119 @@
+# mypy: disable-error-code="override"
```

Comment:
Not a huge fan of this, but happy to leave and fix later if needed. Might be worth seeing if we've inadvertently fixed it in other tidying and can remove it though.

Reply:
It's not removable; it's essentially caused by the attributes being theoretically settable (they're just generic attributes), but the subclass doesn't have setters (because we never change them). There doesn't seem to be a clean way of making those attributes non-settable...

Reply:
This is probably into the realms of the abstract base class stuff, which we _should_ do at some point but for now isn't important so happy to leave this.